src/indexer/services/evmIndexer/ofac/index.ts

```diff
@@ -0,0 +1,34 @@
+import url from "url"
+
+type ChainAnalysisIdentIfication = {
+  category: string
+  name: string
+  description: string
+  url: string
+}
+
+type ChainAnalysisResponse = {
+  identifications: Array<ChainAnalysisIdentIfication> | []
+}
+
+export class OfacComplianceService {
+  private chainAnalisysUrl: string
+  private chainAnalisysApiKey: string
+
+  constructor(chainAnalysisUrl: string, chainAnalisysApiKey: string) {
+    this.chainAnalisysUrl = chainAnalysisUrl
+    this.chainAnalisysApiKey = chainAnalisysApiKey
+  }
+
+  public async checkSanctionedAddress(address: string): Promise<string> {
```

Comment:
This probably should be isAddressSanctioned  and should return bool to be clear.

Reply:
@MakMuftic I remember we talked about this, initially this was going to be a boolean, wdyt?

Reply:
I believe string provides us more flexibility for address to be in more than 2 states, in future we will probably check for multiple "filters".

Reply:
I would at least then make it an enum of available states.

Reply:
That makes sense :+1: 
cc: @wainola

Reply:
ok